### PR TITLE
Script is update according to clarifications.

### PR DESCRIPTION
**Pull request by istoimenova**

### CODE REVIEW

**Comment by istoimenova** [comment]
@VVeremjova please review

File: test_scripts/Polices/user_consent_of_Policies/100_ATF_TC_P_isAppPermissionsRevoked_true.lua [hunk 1]
```diff
@@ -135,7 +135,7 @@ function Test:Precondition_PTU_revoke_app_group()
             {status = "UPDATING"}, {status = "UP_TO_DATE"}):Times(2)
           :Do(function(_,data)
               if(data.params.status == "UP_TO_DATE") then
-                EXPECT_HMINOTIFICATION("SDL.OnAppPermissionChanged", {appID = HMIAppID, isAppPermissionsRevoked = true, appRevokedPermissions = {"DataConsent"}})
+                EXPECT_HMINOTIFICATION("SDL.OnAppPermissionChanged", {appID = HMIAppID, isAppPermissionsRevoked = true, appRevokedPermissions = { {name = "DrivingCharacteristics"} } })
```
Comment (VVeremjova) [inline test_scripts/Polices/user_consent_of_Policies/100_ATF_TC_P_isAppPermissionsRevoked_true.lua]: @istoimenova Please add link for requirements

File: test_scripts/Polices/user_consent_of_Policies/100_ATF_TC_P_isAppPermissionsRevoked_true.lua [hunk 2]
```diff
@@ -148,15 +148,13 @@ function Test:Precondition_PTU_revoke_app_group()
...
-
-  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "NONE", systemContext = "MAIN", audioStreamingState = "NOT_AUDIBLE" })
 end
 
 --[[ Test ]]
 commonFunctions:newTestCasesGroup("Test")
 function Test:TestStep_Activate_app_isAppPermissionRevoked_true()
   local RequestIdActivateAppAgain = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = HMIAppID })
-  EXPECT_HMIRESPONSE(RequestIdActivateAppAgain, { result = { code = 0, method = "SDL.ActivateApp", isAppRevoked = true, isAppPermissionsRevoked = true}})
+  EXPECT_HMIRESPONSE(RequestIdActivateAppAgain, { result = { code = 0, method = "SDL.ActivateApp", isAppRevoked = false, isAppPermissionsRevoked = true, appRevokedPermissions = { {name = "DrivingCharacteristics"} }}})
```
Comment (VVeremjova) [inline test_scripts/Polices/user_consent_of_Policies/100_ATF_TC_P_isAppPermissionsRevoked_true.lua]: @istoimenova Can you provide links for requirements which say that isAppRevoked in this case should be = false?

File: test_scripts/Polices/user_consent_of_Policies/100_ATF_TC_P_isAppPermissionsRevoked_true.lua (hunk 1 above)
Comment (istoimenova) [inline test_scripts/Polices/user_consent_of_Policies/100_ATF_TC_P_isAppPermissionsRevoked_true.lua]: @VVeremjova as requirement is JIRA ID, I can't list it here:
The description: [Policies] SDL.ActivateApp from HMI and 'isAppPermissionsRevoked' parameter in the response

File: test_scripts/Polices/user_consent_of_Policies/100_ATF_TC_P_isAppPermissionsRevoked_true.lua (hunk 2 above)
Comment (istoimenova) [inline test_scripts/Polices/user_consent_of_Policies/100_ATF_TC_P_isAppPermissionsRevoked_true.lua]: @VVeremjova  In defect "[GENIVI] SDL doesn't send appRevokedPermissions in SDL.ActivateApp" we have clarification in comments, please check.

**Pull request merged**